README.md

```diff
@@ -54,7 +54,8 @@ See [contributing](./.github/CONTRIBUTING.md) guidance and the [key decisions lo
 
 This product requires **Java version 1.8** to be installed. Later versions are not supported at present.
 * Clone the repository
-* Follow the steps in the [setup instructions](./generator/README.md). The following need to be actioned for the generator, 
+* Follow the steps in the [setup instructions](./generator/README.md). 
+The following need to be actioned for the generator, 
 the remaining steps need to be actioned to be able to use the profiler.
```

Comment:
Can we remove this line about the profiler, its no longer in the project - didn't realise it was still included.
The preceding line will need to be updated to make sense.

Reply:
Can "The following need to be actioned for the generator" just be removed? I'm not sure what it's trying to convey.

Reply:
I think the updates make sense.